extra/resources/SystemHealth

```diff
@@ -181,7 +181,11 @@ SystemHealth_monitor() {
     if [ $RC -ne 0 ] || [ "$comm" != "$daemon_name" ]; then
         ocf_log debug "${daemon_name} is not running as pid $pid!"
         rm "$pidfile"
-        return $OCF_ERR_GENERIC
+
+        if [ "$__OCF_ACTION" = "monitor" ] && ! ocf_is_probe; then
+            return $OCF_ERR_GENERIC
```

Comment:
I want to point out that I kept `OCF_ERR_GENERIC` here due to the very adamant comment under the `SystemHealth_monitor()` header.

Reply:
Any OCF error code can be used (not NOT_RUNNING, which is not an error -- for a probe, NOT_RUNNING is appropriate, but for a recurring monitor, not running is an error)

Reply:
Making sure we're on the same page: we want an error during a recurring monitor and a NOT_RUNNING for a probe here. Maybe we need a non-NOT_RUNNING code on the recurring monitor, to ensure that a stop gets scheduled to clean everything up.